eventmesh-dashboard-console/src/main/java/org/apache/eventmesh/dashboard/console/entity/client/ClientEntity.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.eventmesh.dashboard.console.entity.client;
+
+import org.apache.eventmesh.dashboard.console.entity.base.BaseEntity;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+
+public class ClientEntity extends BaseEntity {
+
+    private static final long serialVersionUID = 1L;
+
+    @Schema(name = "id", description = "primary key")
+    private Long id;
+
+    private String name;
+
+    private String eventmeshAddress;
```

Comment:
Please edit `Client` class according to SDK docs:
- https://eventmesh.apache.org/zh/docs/sdk-java/http
- https://eventmesh.apache.org/zh/docs/sdk-java/tcp
- https://eventmesh.apache.org/zh/docs/sdk-java/grpc